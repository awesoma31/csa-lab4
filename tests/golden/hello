print("hello world");

inter 1{}
