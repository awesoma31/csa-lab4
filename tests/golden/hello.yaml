instruction_bin: "bin/instr.bin"
data_bin: "bin/data.bin"

max_interruptions: 2
